src/main/webapp/index-script.js

```diff
@@ -391,14 +390,14 @@ function displayReviewModal() {
 function fetchReviews(placeID) {
   const request = '/review?place_id=' + placeID;
   fetch(request).then(response => response.json()).then((place) => {
-    populateReviews(place.reviews, placeID);
+    populateReviews(place.reviews, placeID, place.currUser);
```

Comment:
Warning: Possible merge conflict. No biggie tho

Reply:
We will fix it once it arises :)